DrawerKit/DrawerKit/Internal API/PresentationController+Utilities.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+// Needs to be a free-floating function because it's accessed by extensions of
+// different types.
+func equal(_ lhs: CGFloat, _ rhs: CGFloat) -> Bool {
+    let epsilon: CGFloat = 0.5
+    return abs(lhs - rhs) <= epsilon
+}
+
+// Needs to be a free-floating function because it's accessed by extensions of
+// different types.
+func smallerThanOrEqual(_ lhs: CGFloat, _ rhs: CGFloat) -> Bool {
```

Comment:
Usually this would be called `lessThanOrEqual` unless there is a compelling reason to use `smaller`?

Reply:
I prefer `smaller` because its opposite is `greater` rather than `more`, i.e., to me `smallerThanOrEqual` pairs up with `greaterThanOrEqual` and `lessThanOrEqual` with `moreThanOrEqual` (which I never see used anywhere).

Reply:
Whilst I understand your reasoning (and in terms of English, you are correct), the convention in maths and programming is to use `less than` and `greater than`.

https://english.stackexchange.com/questions/391839/why-is-the-opposite-of-greater-than-less-than

Reply:
But as it's not part of the public API, I can live with it 😄